android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -349,30 +350,37 @@ private void showMainLayout() {
         addRemoveSecureFlag(false);
 
         mCryptoOnboardingLayout.setVisibility(View.GONE);
-        mCryptoLayout.setVisibility(View.VISIBLE);
-
-        mBottomNavigationView.setOnItemSelectedListener(menuItem -> {
-            final int menuItemId = menuItem.getItemId();
-            if (menuItemId == R.id.action_wallet_portfolio) {
-                mViewPager.setCurrentItem(PORTFOLIO_FRAGMENT_POSITION, true);
-            } else if (menuItemId == R.id.action_wallet_activity) {
-                mViewPager.setCurrentItem(TRANSACTIONS_ACTIVITY_FRAGMENT_POSITION, true);
-            } else if (menuItemId == R.id.action_wallet_accounts) {
-                mViewPager.setCurrentItem(ACCOUNTS_FRAGMENT_POSITION, true);
-            } else if (menuItemId == R.id.action_wallet_explore) {
-                mViewPager.setCurrentItem(MARKET_FRAGMENT_POSITION, true);
-            }
-            return true;
-        });
+        if (BraveConfig.WEB_WALLET_ENABLED) {
+            WalletUtils.openWebWallet();
+        } else {
+            mCryptoLayout.setVisibility(View.VISIBLE);
```

Comment:
Maybe early return there?

Reply:
Better keep it as it is, there's some UI logic here to show/hide the crypto layout. Once the WebUI implementation will be fully integrated and the build flag removed we can remove all the unused code.

Reply:
I'm asking for this:
```
if (BraveConfig.WEB_WALLET_ENABLED) {
    WalletUtils.openWebWallet();
    return;
}
```
So no extra scope. This is easier to read and easier to review now and when feature gets inlined.

Reply:
As a general rule of thumb, the Android codebase uses early returns when some kind of validation is not satisfied, e.g. `if (targetActivity == null) return;` or `if (size == 0) return;`.  In all the other cases instead we use `if, else`. 

It's probably a matter of style, I personally find it easier to read as it's more consistent with the actual codebase. My preference leans towards `if, else`, so if it doesn't bother you too much I would stick to it. 

Otherwise I have no issue changing it, they are both logically correct. But you'll find the `if..else` style pretty much everywhere in our codebase. It's also the style adopted by Google; here is a random page from the official documentation and is farly new (graph API has been recently introduced) https://developer.android.com/guide/navigation/use-graph/conditional

<img width="947" alt="Conditional_navigation_ _ _Android_Developers" src="https://github.com/brave/brave-core/assets/3308503/820553ba-0539-4d25-b2a9-0c3a768886c1">

<img width="958" alt="Conditional_navigation_ _ _Android_Developers" src="https://github.com/brave/brave-core/assets/3308503/6295b900-f063-45d3-93f1-582de934ccc9">